src/shared/components/query/CaseSetSelectorUtils.ts

```diff
@@ -23,17 +23,27 @@ export const CustomCaseSets: CustomCaseSet[] = [
 	{name: 'User-defined Case List', description: 'Specify your own case list', value: CaseSetId.custom, isdefault : true}
 ]
 
-export function getFilteredCustomCaseSets(isVirtualStudy:boolean, profiledSamplesCount: { w_mut: number; w_cna: number; w_mut_cna: number; all: number}) {
+export function getFilteredCustomCaseSets(isVirtualStudy:boolean, isMultipleNonVirtualStudies: boolean, profiledSamplesCount: { w_mut: number; w_cna: number; w_mut_cna: number; all: number}) {
```

Comment:
I think `isVirtualStudy` and `isMultipleNonVirtualStudies` can be combined to one parameter

Reply:
@kalletlak But how could we distinguish between the single physical study and multiple physical studies? Because we don't want to add additional `CustomCaseSets` to the single physical study.

Reply:
okay